### ISSUE TITLE
Make the style look similar to ChatGPT

### ISSUE THREAD

**Issue by jcubic** [jcubic/chat-gpt#38]
This is the ChatGPT interface:

![Przechwycenie obrazu ekranu_2024-09-09_23-32-58](https://github.com/user-attachments/assets/21c8fdff-7ffe-4bdc-b0fe-7eded1a3a2fe)

This is the current style:

![Przechwycenie obrazu ekranu_2024-09-09_23-34-09](https://github.com/user-attachments/assets/f8b9eeb7-35bc-492f-b7e2-254da6125cbc)

It would be nice to make the use the same.

The task is to modify CSS and put it into generate HTML in the bookmark.js file.

Code snippets are tracked in different issue:
* https://github.com/jcubic/chat-gpt/issues/35

**Comment by jcubic** [comment]
Added Hacktoberfest label, you don't need to be assigned to an issue to work on it. I never heard anyone use assignments for people that are not part of the core team.

I've only seen people from Hacktoberfest asking to assign them to issues. Do you have instructions to do this?

**Comment by cypher4802** [comment]
Not really necessity but it create sense of security that this task is assigned to me

**Comment by cypher4802** [comment]
Can you pls elaborate a bit do you want to style all particular chats or just the 
![Screenshot 2024-10-24 021605](https://github.com/user-attachments/assets/866a6d8a-eaf6-4686-898f-72f153fb50bc)
list I have styled the list(ss in attachment).

**Comment by jcubic** [comment]
No, the task is to modify the bookmarklet to generate the code that look similar to the output of ChatGPT as on the screenshot. And creating example conversation with current date that show how it works.

**Comment by shinigami-777** [comment]
@jcubic Can I work on this issue

**Comment by jcubic** [comment]
@shinigami-777 yes
